modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/BreadcrumbFinder/index.tsx

```diff
@@ -41,14 +78,17 @@ const BreadcrumbFinder = () => {
 			className="breadcrumb-finder-container breadcrumb-finder-navigator"
 			id="breadcrumbFinderContainer"
```

Comment:
Maybe remove this id

```suggestion
			className="breadcrumb-finder-container breadcrumb-finder-navigator"
			id="breadcrumbFinderContainer"
```